apps/cowswap-frontend/src/common/analytics/types.ts

```diff
@@ -0,0 +1,56 @@
+import { AnalyticsCategory, Category, GtmEvent, toGtmEvent } from '@cowprotocol/analytics'
+
+/**
+ * CowSwap-specific analytics categories
+ * Extends the base Category enum with CowSwap-specific values
+ */
+export enum CowSwapCategory {
+  // Base Category values (must match exactly)
+  SERVICE_WORKER = Category.SERVICE_WORKER,
+  FOOTER = Category.FOOTER,
+  EXTERNAL_LINK = Category.EXTERNAL_LINK,
+
+  // Trade Categories
+  TRADE = 'Trade',
+  LIST = 'Lists',
+  HOOKS = 'Hooks',
+  CURRENCY_SELECT = 'Currency Select',
+  RECIPIENT_ADDRESS = 'Recipient address',
+  ORDER_SLIPAGE_TOLERANCE = 'Order Slippage Tolerance',
+  ORDER_EXPIRATION_TIME = 'Order Expiration Time',
+  WRAP_NATIVE_TOKEN = 'Wrapped Native Token',
+  CLAIM_COW_FOR_LOCKED_GNO = 'Claim COW for Locked GNO',
+  TWAP = 'TWAP',
+  SURPLUS_MODAL = 'Surplus Modal',
+  COWSWAP = 'CoWSwap',
+  LIMIT_ORDER_SETTINGS = 'Limit Order Settings',
+
+  // UI Categories
+  THEME = 'Theme',
+  WALLET = 'Wallet',
+  NOTIFICATIONS = 'Notifications',
+  PROGRESS_BAR = 'Progress Bar',
+  GAMES = 'Games',
+  INIT = 'Init',
+  COW_FORTUNE = 'CoWFortune',
+}
+
+// Type assertion to ensure CowSwapCategory extends AnalyticsCategory
+const _assertType: AnalyticsCategory = CowSwapCategory.SERVICE_WORKER
```

Comment:
The assertion looks redundant. The value implicitly inherited from AnalyticsCategory: `SERVICE_WORKER = Category.SERVICE_WORKER,`

Reply:
Removed